microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -190,15 +131,98 @@ You can access Collections from an InPrivate window when using a signed-out prof
+
+The Compose tool might not be available on some webpages or for certain input fields.
+
+Text that you've highlighted or inputted are sent to Microsoft only after you click the **Rewrite** or **Generate** button, not while you are selecting text.  No additional webpage data is ever shared.  Requests for Compose are sent over a secure HTTPS connection.  Each request contains the input prompt text, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.
+
+To disable the Compose tool for all webpages, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages**, and then turn off the toggle **Collaborate with Copilot as you write on the web**.
+
+For enterprises, additional options for data protection are available for Compose.  See [Privacy and protections](/copilot/privacy-and-protections) in the Microsoft Copilot documentation.
+
+
+<!-- ====================================================================== -->
+## Copilot in Microsoft Edge sidebar
+
+In Copilot in the Microsoft Edge sidebar, you can ask complex questions, find comprehensive answers, get summarized information, and find inspiration to build upon, in a side-by-side view, with no need to flip between tabs.
+
+To open Copilot in the Microsoft Edge sidebar, click the **Copilot** (![Copilot icon.](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of the Microsoft Edge toolbar: 
+
+![Summarize this page](./index-images/summarize-content.png) 
+
+
+<!-- --------------------------------------------------------------------- -->
+#### Data and consent used by Copilot in Microsoft Edge 
+
+Microsoft Edge determines what data to send to Copilot based on the user's **query** and their consent to share data with Microsoft.  For questions that don't need **browsing context**, such as "Help me plan a trip to Cannon Beach", Microsoft Edge shares the URL, page title, user's query, and previous conversation history to help Copilot answer their query effectively.
+
+For questions that need **browsing context**, such as when the user asks Copilot to summarize a large page of text, Microsoft Edge requests permission from the user to access page information:
+
+![Copilot asking for consent to access the page content in Microsoft Edge](./index-images/consent-for-contextual-browsing.png) <!-- Update Bing chat screenshot-->
+
+When the user grants permission to share page information, Microsoft Edge sends the full **browsing context** in addition to the user's **query** and previous **conversation history** to Copilot, to help generate a meaningful **response**.
+
+The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle.
```

Comment:
as an option, corp style, not necessarily binding:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/toggle

Reply:
seems like section needs to change "the user" to "you" per corp style

Reply:
rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-3206#data-and-consent-used-by-copilot-in-microsoft-edge

Reply:
Survey of use of 'toggle' in whitepaper shows need for global correction of usage of 'toggle', to match corp style & for consistency, to make maintenance easier: this survey covers "A" through "Image descriptions":
* Toggle the **Show me search and site suggestions using my typed characters** setting. - incorrect per corp style.
* Select **Personal info** and toggle the **Save and fill personal info** setting. - incorrect.
* Toggle off the **Offer to save passwords** setting.  - incorrect.
* Go to `edge://settings/profiles/sync` and turn off the **Basic info** toggle. - correct (other than special url instead of GUI nav path).
* If you want to disable Collections for a signed-in profile using a personal account, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**. - correct.
* To disable the Compose tool for all webpages, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Languages**, and then turn off the toggle **Collaborate with Copilot as you write on the web**. - correct 
* The user can modify this permission anytime by going to Microsoft Edge > **Settings** > **Sidebar** > **App and notification settings** > **App specific settings** > **Copilot** and then turning on or off the **Allow access to any webpage or PDF** toggle. - correct except "the user" should be "you", and should "to do x, turn ..." not "by turning".
* Toggle the **Allow sites to play protected content (recommended)** and **Allow identifiers for protected content (computer restart may be required)** settings. - incorrect.
* To change SmartScreen settings, go to `edge://settings/privacy` and toggle SmartScreen. - incorrect, and "SmartScreen" should be bold & match UI label.
* Turn off the **From other sources** toggle to uninstall extensions. - ok but sentence should be reversed: To do x, turn off.
* Toggle the **Allow extensions from other stores** setting. - incorrect.
* When the **Include related matches in Find on page** toggle is turned on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection.  - correct.
* 1. Turn off the **Include related matches in Find on page** toggle. - correct.
* Or, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Save screenshots of site for History**.  - correct (except special url should be GUI nav path).
* Toggle the **Get image descriptions from Microsoft for screen readers** setting.  - incorrect.

Reply:
result re: the Suggestion:

```
When you grant permission to share page information, Microsoft Edge sends the 
full **browsing context** in addition to your **query** and previous **conversation history** 
to Copilot, to help generate a meaningful **response**.

To modify this permission, go to Microsoft Edge > **Settings** > **Sidebar** > **App 
and notification settings** > **App specific settings** > **Copilot**, and then turn on or 
off the **Allow access to any webpage or PDF** toggle.
```